seq2science/envs/pytxi.yaml

```diff
@@ -7,4 +7,4 @@ dependencies:
   - genomepy=0.11.1
   - pip
   - pip:
-    - git+https://github.com/vanheeringen-lab/pytxi.git@v0.1.1
+    - git+https://github.com/vanheeringen-lab/pytxi.git@develop
```

Comment:
Maybe we put it on bioconda?

Reply:
I might when it's stable. Had to change it yesterday due to the pyproject.toml structure changes

Reply:
Yeah but this might break future seq2science runs because develops keep changing. Maybe you can pin it on a commit instead of a branch?

Reply:
Definetly! I'll pin it before merging.

For some reason, genomepy 0.12 is still not available on bioconda (I think there is an issue with some of the dependencies), so I can't test this PR in a clean environment yet.